shapeworks_cloud/core/deepssm_tasks.py

```diff
@@ -0,0 +1,557 @@
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Dict
+
+import DataAugmentationUtils
+import DeepSSMUtils
+from celery import shared_task
+from django.conf import settings
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+import shapeworks as sw
+
+from shapeworks_cloud.core import models
+from swcc.api import swcc_session
+from swcc.models import Project as SWCCProject
+
+from .tasks import edit_swproj_section
+
+
+def interpret_deepssm_form_data(data: Dict) -> Dict:
+    # Prep
+    data['validation_split'] = data['validationSplit']
+    data['testing_split'] = data['testingSplit']
+    data['image_spacing'] = data['imageSpacing']
+    data['percent_variability'] = data['percentVariability']
+
+    # Augmentation
+    data['aug_num_samples'] = data['numSamples']
+    data['aug_sampler_type'] = data['samplerType']
+
+    # Training
+    data['train_loss_function'] = data['lossFunction']
+    data['train_epochs'] = data['epochs']
+    data['train_learning_rate'] = data['learningRate']
+    data['train_batch_size'] = data['batchSize']
+    data['train_decay_learning_rate'] = data['decayLearningRate']
+    data['train_fine_tuning'] = data['fineTuning']
+    data['train_fine_tuning_epochs'] = data['ftEpochs']
+    data['train_fine_tuning_learning_rate'] = data['ftLearningRate']
+
+    return data
+
+
+def run_prep(params, project, project_file, progress):
+    # //////////////////////////////////////////////
+    # /// STEP 1: Create Split
+    # //////////////////////////////////////////////
+    val_split = float(params['validation_split'])
+    test_split = float(params['testing_split'])
+    # val_split = project.get_parameters('validation_split')
+    # test_split = project.get_parameters('testing_split')
+    train_split = 100.0 - val_split - test_split
+    DeepSSMUtils.create_split(project, train_split, val_split, test_split)
+
+    # /////////////////////////////////////////////////////////////////
+    # /// STEP 2: Groom Training Shapes
+    # /////////////////////////////////////////////////////////////////
+    params = project.get_parameters('groom')
```

Comment:
Good catch. This was meant to be a temporary object which lets us set parameters for the grooming and optimization tasks within the sub-task. These parameters are for groom or optimization, and separate from the formData params.

I think it was only working because there weren't references to the `params['x']` after these reassignments.

Reply:
Fixed: [0c0b918](https://github.com/girder/shapeworks-cloud/pull/337/commits/0c0b9182bb5416c8e26357f8d48f9ee5e18e30d0)